guardian/screeny
# Support for interactive migration

## What does this change?
This PR updates screeny for use as part of the interactive migration. The primary change is to allow a list of URLs to be read from a Google sheet. The result of running the application is unchanged: pngs showing a side-by-side comparison of pages rendered via DCR compared to Frontend.

## How to test
Given a valid key from url box and (if applicable) valid credentials for access to Google APIs, then run the application and expect to see the resulting pngs in `./screenshots`. Each png should contain 2 views of the same page: left hand side = rendered via DCR, right hand side = rendered via Frontend.

## How can we measure success?
Pages are rendered successfully, pngs are saved to screenshots directory and are then available for manual comparison.
